File: tests/fixtures/v2-process-flow/positive/process_flow.go
package processflow

func direct(r *Request) {
	exec.Command(r.FormValue("direct"))
}

func propagated(r *Request) {
	command := r.FormValue("propagated")
	alias := command
	exec.Command(alias)
}

func separated(r *Request) {
	exec.Command("tool", r.FormValue("argument"))
}

File: tests/fixtures/v2-process-flow/positive/ProcessFlow.java
class ProcessFlow {
    void direct(HttpServletRequest request) throws Exception {
        Runtime.getRuntime().exec(request.getParameter("direct"));
    }

    void propagated(HttpServletRequest request) throws Exception {
        String command = request.getParameter("propagated");
        String alias = command;
        Runtime.getRuntime().exec(alias);
    }

    void separated(HttpServletRequest request) {
        new ProcessBuilder("tool", request.getParameter("argument"));
    }
}
